### DISCUSSION TITLE
Zulassungen Spanien

### DISCUSSION THREAD

**Discussion by C1710** [marhei/TrainCompany-Data#326] [Q&A] [answered]
Soll ich TGV Atlantique auch für Spanien zulassen, weil der AVE 100 darauf basiert oder lieber separat einfügen/nur Euroduplex erlauben?

Ich denke mal, 403/406 als Velaro ES wäre seeehr gewagt 😂

**Comment by marhei** [comment, marked as answer]
Denke eher als was seperates und dem 2N2 noch Spanienzulassung geben.
